src/Canvas.js

```diff
@@ -39,7 +41,7 @@ class Canvas {
     this._foreground = {r: -1, g: -1, b: -1};
     this._display = {bold: false, dim: false, underlined: false, blink: false, reverse: false, hidden: false};
 
-    this._cells = Array.from({length: width * height}).map(() => new Cell());
+    this._cells = Array.from({length: width * height}).map(() => new Cell(undefined, { colorSupport }));
```

Comment:
Why `undefined`, each cell must have some character, instead there are risks to make it working much slower